lib/mining-utils.js

```diff
@@ -9,7 +9,14 @@ const createKeccakHash = require('keccak');
 const CPV_JUMP_FACTOR = 7;
 const CPV_JUMP = 64;
 const MAX_NUMBER_OF_UNCLES = 3;
-
+const host = config.rskd.url + ":" + config.rskd.rpcport;
```

Comment:
`defaultContext`'s `Host:` header gets populated with this, and then used to call a Bitcoin's RPC, not RSK one. You should either use `config.bitcoind.*` in here, or create a `defaultBTCContext` with `config.bitcoind.*` and use that one in the added RPC calls.

Reply:
fixed, thanks